cmd/utils/customflags.go

```diff
@@ -47,10 +47,7 @@ func (self *DirectoryString) Set(value string) error {
 // e.g. ~/.ethereum -> /home/username/.ethereum
 type DirectoryFlag struct {
 	cli.GenericFlag
-	Name   string
-	Value  DirectoryString
-	Usage  string
-	EnvVar string
```

Comment:
Fun fact, these have always been part of `cli.GenericFlag`, but because they have also been declared outside, some functionality of the `cli` package didn't work properly for them. E.g. listing all allowed flags never returned instances of  `DirectoryFlag` because the `Name` was empty (as the interface methods of `cli.GenericFlag` works on the data elements of `cli.GenericFlag` and not "overrides" declared in `DirectoryFlag`.